src/GitHub.VisualStudio/UI/Views/PullRequestCreationView.xaml.cs

```diff
@@ -17,7 +17,8 @@ public partial class PullRequestCreationView : GenericPullRequestCreationView
         public PullRequestCreationView()
         {
             InitializeComponent();
-            DataContextChanged += (s, e) => ViewModel = e.NewValue as IPullRequestCreationViewModel;
+            // DataContextChanged += (s, e) => ViewModel = e.NewValue as IPullRequestCreationViewModel;
+            DataContextChanged += (s, e) => ViewModel = new GitHub.SampleData.PullRequestCreationViewModelDesigner() as IPullRequestCreationViewModel;
```

Comment:
You can kill this change, it doesn't really do anything valuable and it'll break things when we hook up real data.